core/src/main/java/greencity/controller/UserController.java

```diff
@@ -244,35 +255,36 @@ public ResponseEntity<UserUpdateDto> updateUser(@Valid @RequestBody UserUpdateDt
      */
     @Operation(summary = "Get available custom shopping list items for current user.")
     @ApiResponses(value = {
-        @ApiResponse(responseCode = "200", description = HttpStatuses.OK),
-        @ApiResponse(responseCode = "400", description = HttpStatuses.BAD_REQUEST),
-        @ApiResponse(responseCode = "401", description = HttpStatuses.UNAUTHORIZED),
+            @ApiResponse(responseCode = "200", description = HttpStatuses.OK),
+            @ApiResponse(responseCode = "400", description = HttpStatuses.BAD_REQUEST),
+            @ApiResponse(responseCode = "401", description = HttpStatuses.UNAUTHORIZED),
     })
     @GetMapping("/{userId}/{habitId}/custom-shopping-list-items/available")
     public ResponseEntity<List<CustomShoppingListItemResponseDto>> getAvailableCustomShoppingListItems(
-        @Parameter(description = "Id of current user. Cannot be empty.") @PathVariable @CurrentUserId Long userId,
-        @PathVariable Long habitId) {
+            @Parameter(description = "Id of current user. Cannot be empty.") @PathVariable @CurrentUserId Long userId,
+            @PathVariable Long habitId) {
+
         return ResponseEntity
-            .status(HttpStatus.OK)
-            .body(userService.getAvailableCustomShoppingListItems(userId, habitId));
+                .status(HttpStatus.OK)
+                .body(userService.getAvailableCustomShoppingListItems(userId, habitId));
     }
 
     /**
-     * Counts all users by user {@link UserStatus} ACTIVATED.
+     * Counts all users by user {@link UserStatus} ACTIVATEDd
```

Comment:
Guess. You copied code into this class from your other branch cause it contains the same code and the same errors.